app/reports/mi-report/get-events.js

```diff
@@ -0,0 +1,17 @@
+const { PAYMENT_EVENT } = require('../../constants/event-types')
+const { getClient, odata } = require('../../storage')
```

Comment:
What's the significance of odata?

Reply:
Querying rows in Azure Table Storage follows odata spec -> https://www.odata.org/

The Azure SDK handily includes a helper method you can pass your `odata` queries too.  So we require this and you'll see on line `6` we've used it for the filter.

Reply:
Thanks!